### PR TITLE
SEO-3147 JS Desc too long short

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,

Please review the changes for JS Desc too long short.

Regards,
Yvone.

### CODE REVIEW

File: js/DateRangePicker/display-format.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+title: Display Format in JavaScript DateRangePicker control | Syncfusion
+description: Learn here more about display format support in Syncfusion Essential JavaScript DateRangePicker Control, its elements, and more.
 platform: js
 control: DateRangePicker
 documentation: ug
 api: /api/js/ejdaterangepicker
 ---
 
-## Display Format
+# Display Format in Essential JavaScript DateRangePicker
```
Comment (Aishwarya-Ganesan) [inline js/DateRangePicker/display-format.md:11]: Remove Essential word.

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Commented.

File: js/DateTimePicker/Overview.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+title: Overview in JavaScript DateTimePicker control | Syncfusion
+description: Learn here more about Overview in Syncfusion Essential JavaScript DateTimePicker Control, its elements, and more.
 platform: js
 control: DateTimePicker
 documentation: ug
 api: /api/js/ejdatetimepicker
 ---
 
-# Overview
+# Overview in Essential JavaScript DateTimePicker
```
Comment (gowthamanthavasiyappan) [inline js/DateTimePicker/Overview.md:11]: Remove Essential word.

File: js/Grid/Overview.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-description: Overview for Grid
+title: Grid widget for Syncfusion Essential JS
+description: Learn here more about Overview in Syncfusion Essential JavaScript Grid Control, its elements, and more.
 platform: js
 control: Grid
 documentation: ug
 api: /api/js/ejgrid
 ---
-# Overview
+# Overview in Essential JavaScript Grid
```
Comment (gowthamanthavasiyappan) [inline js/Grid/Overview.md:10]: Remove Essential word.

File: js/ReportDesigner/Compose-Report/Configure-ReportItem/Line-ReportItem.md [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
+title: Line Report Item in JavaScript ReportDesigner control | Syncfusion
+description: Learn here more about Draw a Line Item support in Syncfusion Essential JavaScript ReportDesigner Control, its elements, and more.
 platform: js
 control: ReportDesigner
 documentation: ug
 api: /api/js/ejreportdesigner
 ---
 
-# Line
+# Line in Essential JavaScript ReportDesigner
```
Comment (gowthamanthavasiyappan) [inline js/ReportDesigner/Compose-Report/Configure-ReportItem/Line-ReportItem.md:11]: Remove Essential word.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]

**Comment by gowthamanthavasiyappan** [comment]
Commented

File: js/ReportDesigner/Compose-Report/Configure-ReportItem/Line-ReportItem.md (hunk 4 above)
Comment (Yvone-Atieno) [reply js/ReportDesigner/Compose-Report/Configure-ReportItem/Line-ReportItem.md]: removed

File: js/Grid/Overview.md (hunk 3 above)
Comment (Yvone-Atieno) [reply js/Grid/Overview.md]: removed

File: js/DateTimePicker/Overview.md (hunk 2 above)
Comment (Yvone-Atieno) [reply js/DateTimePicker/Overview.md]: removed

**Review by gowthamanthavasiyappan — APPROVED** [review]

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**